source/shared/cpp/ObjectModel/TextBlock.h

```diff
@@ -61,5 +62,6 @@ class TextBlock : public BaseCardElement
     bool m_wrap;
     unsigned int m_maxLines;
     HorizontalAlignment m_hAlignment;
+    std::string ParseISO8601() const;
```

Comment:
>ParseISO8601 [](start = 16, length = 12)

let's call this ParseDateTime. the ISO is no longer accurate

Reply:
thanks

---
In reply to: [150950502](https://github.com/Microsoft/AdaptiveCards/pull/871#discussion_r150950502) [](ancestors = 150950502)